src/commands/help.js

```diff
@@ -1,8 +1,19 @@
+const billhelp = require('./bill/help');
+const confighelp = require('./config/help');
+
 exports.handle = function(_args, message, _client) {
-  let help = 'Here is a list of commands:\n';
-  help += 'Type `!ping`.\n';
-  help += 'Type `!help` for general help.\n';
-  help += 'Type `!bill help` for help regarding bills.\n';
-  help += 'Type `!config help` for help regarding configuration.';
-  message.reply(help);
+  if (_args == 'bill') {
```

Comment:
If you are using an argument, remove the underscore. The underscore means the argument is not used

Reply:
Also, _args is a list. It will never assume the value 'bill' but args[0] could be "bill" or maybe args = ["bill"]

Reply:
I thought the underscore meant that the property was private.
Sorry this is because some naming conventions in other languages has made it hard for me to read underscore differently in front of a property or argument.